web/src/portico/team.ts

```diff
@@ -103,11 +132,16 @@ export default function render_tabs(contributors) {
 
         $(`#${CSS.escape(tab_name)}`).on("click", () => {
             if (!loaded_repos.includes(repo_name)) {
-                const filtered_by_repo = contributors_list.filter((c) => c[repo_name]);
+                const filtered_by_repo = contributors_list.filter((c) => c[repo_name] || 0);
                 const html = filtered_by_repo
-                    .sort((a, b) =>
-                        a[repo_name] < b[repo_name] ? 1 : a[repo_name] > b[repo_name] ? -1 : 0,
-                    )
+                    .sort((a, b) => {
+                        const a_commits = a[repo_name];
```

Comment:
Did you consider doing `= a[repo_name] || 0` for these references? It might be easier to think about.

Reply:
That is also a way to fix the `undefined` problem. Both are good options IMO. But the one you mentioned is shorter and more intuitive, so pushing that one. Thanks for the suggestion. :heart: